dashboard/client/src/common/ServeStatus.component.test.tsx

```diff
@@ -0,0 +1,76 @@
+import { render, screen } from "@testing-library/react";
+import React from "react";
+import { MemoryRouter } from "react-router-dom";
+import { ServeApplication, ServeApplicationStatus } from "../type/serve";
+import { ServeStatusIcon } from "./ServeStatus";
+
+const APP: ServeApplication = {
+  name: "MyServeApp",
+  route_prefix: "/my-serve-app",
+  docs_path: null,
+  status: ServeApplicationStatus.RUNNING,
+  message: "",
+  last_deployed_time_s: 1682029771.0748637,
+  deployed_app_config: null,
+  deployments: {},
+};
+
+describe("ServeStatusIcon", () => {
+  it("renders RUNNING status", async () => {
+    await render(<ServeStatusIcon app={APP} small={false} />, {
+      wrapper: MemoryRouter,
+    });
+    const icon = screen.getByTestId("serve-status-icon");
```

Comment:
please use `await screen.findByText` after render for all tests

Reply:
Sounds good to me! 

I used screen.findByText in RecentServeCard.component.test

I directly verified the ServeStatusIcon in the ServeStatus.component.test since it is a simply component to render.

Reply:
Let's do findBy for all cases just to be extra safe. It's also simpler if we say there is a pattern we always do `findBy` after render. It's hard for someone to know what a "Simple" component to render means. And if the component becomes non-simple in the future, the tests do not start breakingn.

Reply:
findBytextID